pages/publish-nft-book/index.vue

```diff
@@ -70,6 +75,8 @@ const hasFiles = computed(() => {
 const shouldDisableNext = computed(() => {
   if (step.value === 0) {
     return uploadFormRef.value?.uploadStatus !== ''
+  } else if (step.value === 1) {
+    return registerISCN.value?.isFormValid !== true
```

Comment:
Can we simplify this?
```suggestion
    return !registerISCN.value?.isFormValid
```

Reply:
[🎨 Improve nextStep logic](https://github.com/likecoin/nft-book-press/pull/292/commits/0af8854eb74fd97ade60972d8ee76ee1267ffa2d)